tests/spark/test_spark_connect_model_export.py

```diff
@@ -0,0 +1,146 @@
+import json
+import os
+
+import numpy as np
+import pandas as pd
+import pyspark
+import pytest
+from packaging.version import Version
+
+if Version(pyspark.__version__) < Version("3.5"):
+    pytest.skip("pyspark ML connect Model is available in pyspark >= 3.5")
```

Comment:
Can we move this after import statements?

Reply:
It has to be before `from pyspark.ml.connect...` statements

Reply:
Can we use lazy imports?